src/shared/bus-polkit.c

```diff
@@ -397,17 +412,16 @@ static int async_polkit_query_check_action(
         assert(q);
         assert(action);
 
-        LIST_FOREACH(authorized, a, q->authorized_actions)
-                if (streq(a->action, action) && strv_equal(a->details, (char**) details))
-                        return 1; /* Allow! */
+        if (async_polkit_query_have_action(q, action, details))
+                return true; /* Allow! */
```

Comment:
Hmm, here you change this from `1` to `true`, and later changed things back? Might be a mistake?